common/src/main/java/com/wynntils/services/mapdata/type/MapPath.java

```diff
@@ -5,9 +5,15 @@
 package com.wynntils.services.mapdata.type;
 
 import com.wynntils.utils.mc.type.Location;
+import com.wynntils.utils.type.BoundingBox;
 import java.util.List;
 
 public interface MapPath extends MapFeature {
     // The path is described by a sequence of locations
     List<Location> getPath();
+
+    @Override
+    default boolean isVisible(BoundingBox boundingBox) {
+        return getPath().stream().anyMatch(location -> boundingBox.contains(location.x(), location.z()));
```

Comment:
Same here, the path might stretch across the bounding box (from e.g. down of it to up of it).